internal/command/org_member.go

```diff
@@ -145,7 +145,8 @@ func (c *Commands) RemoveOrgMember(ctx context.Context, orgID, userID string) (*
 		return nil, err
 	}
 	if errors.IsNotFound(err) {
-		return nil, nil
+		// empty response because we have no data that match the request
+		return &domain.ObjectDetails{}, nil
```

Comment:
Wouldn't an is not found error be more consistent?

Reply:
It would but break the API behaviour from previous versions

Reply:
Also, the test TestCommandSide_RemoveIAMMember/member_not_existing,_nil_result fails now @adlerhurst

Reply:
I changed expected return values in the tests